<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="m_central_widget"/>
  <widget class="QMenuBar" name="m_menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_menu_file">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="m_menu_open_recent">
     <property name="title">
      <string>Open &amp;Recent</string>
     </property>
     <addaction name="separator"/>
     <addaction name="m_action_clear_recent"/>
    </widget>
    <widget class="QMenu" name="m_menu_new">
     <property name="title">
      <string>&amp;New</string>
     </property>
     <addaction name="m_action_new_project"/>
    </widget>
    <addaction name="m_menu_new"/>
    <addaction name="m_action_open"/>
    <addaction name="m_menu_open_recent"/>
    <addaction name="m_action_close_project"/>
    <addaction name="separator"/>
    <addaction name="m_action_settings"/>
    <addaction name="separator"/>
    <addaction name="m_action_save"/>
    <addaction name="m_action_save_all"/>
    <addaction name="separator"/>
    <addaction name="m_action_exit"/>
   </widget>
   <widget class="QMenu" name="m_menu_edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="m_action_undo"/>
    <addaction name="m_action_redo"/>
    <addaction name="separator"/>
    <addaction name="m_action_cut"/>
    <addaction name="m_action_copy"/>
    <addaction name="m_action_delete"/>
   </widget>
   <widget class="QMenu" name="m_menu_view">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="m_menu_views_and_toolbars">
     <property name="title">
      <string>Views and Toolbars</string>
     </property>
    </widget>
    <addaction name="m_menu_views_and_toolbars"/>
   </widget>
   <widget class="QMenu" name="m_menu_help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="m_action_about"/>
   </widget>
   <addaction name="m_menu_file"/>
   <addaction name="m_menu_edit"/>
   <addaction name="m_menu_view"/>
   <addaction name="m_menu_help"/>
  </widget>
  <widget class="QStatusBar" name="m_status_bar"/>
  <action name="m_action_clear_recent">
   <property name="text">
    <string>Clear Recent</string>
   </property>
  </action>
  <action name="m_action_new_project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="m_action_open">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="m_action_close_project">
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="m_action_settings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="m_action_save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="m_action_save_as">
   <property name="text">
    <string>&amp;Save As</string>
   </property>
  </action>
  <action name="m_action_save_all">
   <property name="text">
    <string>&amp;Save All</string>
   </property>
  </action>
  <action name="m_action_exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="m_action_undo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="m_action_redo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="m_action_cut">
   <property name="text">
    <string>&amp;Cut</string>
   </property>
  </action>
  <action name="m_action_copy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="m_action_delete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="m_action_about">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
